src/components/OpacityView.js

```diff
@@ -21,7 +21,7 @@ const propTypes = {
      * @default []
      */
     // eslint-disable-next-line react/forbid-prop-types
-    style: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+    style: PropTypes.oneOfType([PropTypes.object, PropTypes.array, PropTypes.bool]),
```

Comment:
maybe  we can use type from [`styles/stylePropTypes.js`](https://github.com/Expensify/App/blob/b321b0a178f93300ba52d278f31cf0723018b88b/src/styles/stylePropTypes.js) ?

Reply:
Done, but I had to update `stylePropTypes`